content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -0,0 +1,59 @@
+---
+title: Guide to Plugin Hosting
+layout: developerguide
+---
+
+This guide will cover the entire plugin hosting process.
+
+Once you've completed all steps described on this page you will have accomplished the following:
+
+* Your plugin source code will be hosted in a repository in the +jenkinsci+ GitHub organization.
+  You will have admin access to this repository.
+* You will be allowed to release the plugin to the Jenkins project Maven repository, which serves as a source for the Jenkins project operated update sites.
+
+== Preparation
+
+Complete the following steps before requesting plugin hosting with the Jenkins project:
+
+. Review the link:../preparation[preparation steps] and make sure they're satisfied.
+. Make sure your plugin follows the plugin naming convention outlined in the link:../style-guides[style guides].
+. Have a public repository containing the plugin source code on GitHub.
+
+
+== Open HOSTING request
+
+Log in to link:https://issues.jenkins-ci.org/[JIRA] and create a new issue link:https://issues.jenkins-ci.org/browse/HOSTING[in the HOSTING] project.
+Make sure to fill out all fields as described.
+
+A Jenkins project member will review your request within a few days.
+If any changes are requested, please implement them.
+
+Once all requirements have been satisfied, your repository will be forked into the +jenkinsci+ organization and you will be invited to join it, and a JIRA component in the JENKINS project will be created for you.
+
+
+== Create a Wiki Page
+
+While not strictly necessary to release your plugin, it's a good idea to create a wiki page for your plugin to store documentation.
+See link:../wiki-page[the guide to plugin wiki pages] for details how to do this.
+
+
+== Enable CI Builds
+
+The Jenkins project hosts a Jenkins instance to perform continuous integration builds for plugins.
+We recommend you set up CI builds for your plugin in the +jenkinsci+ GitHub organization by creating a +Jenkinsfile+ in your plugin's GitHub repository.
+See link:../continuous-integration[the documentation for CI builds] for details how to do this.
+
+
+== Request Upload Permissions
+
+After your plugin source code has been forked into the +jenkinsci+ organization, you will need to file a request for upload permissions.
+link:https://github.com/jenkins-infra/repository-permissions-updater/[Follow the instructions in the README of this repository] to do that.
+
+Once you've completed this step and your request has been merged, you will be able to link:../releasing/[release your plugin].
+See that guide for release instructions.
```

Comment:
I guess you want(ed) to add a link to _../releasing.adoc_ didn't you?

Reply:
Previous line/sentence.